### ISSUE TITLE
About released files on NPM

### ISSUE THREAD

**Issue by pvnr0082t** [wrangr/psl#2]
Hi @lupomontero ,
I'm a member of [cdnjs](https://github.com/cdnjs/cdnjs) team and we want to host this library.
I found that the files published on NPM are different from GitHub repository. 
Files on NPM looks like as follow:
```js
.
├── data
│   └── rules.json
├── index.js
├── karma.conf.js
├── package.json
└── README.md
```
I think it's better to get main files from GitHub. Or will the files under `dist` folder be published on NPM in the future? This will affect our decision using `git` or `npm` auto-update config.
Thank you.

https://github.com/cdnjs/cdnjs/issues/8942

**Comment by lupomontero** [comment]
Hi @pvnr0082t, thanks for opening this issue. It'd be great to see this library on cdnjs ;-)

The reason the files are different is because the files in `dist/` are only used when the library is used in the browser, and the main `index.js` is used when required in Node.js (or browserify). There are basically two entry points.

What would you suggest as a solution? I could simply remove the `dist/` directory from `.npmignore` to include it on NPM, but the `main` property in the `package.json` would still point to `index.js` (which won't work if included directly in the browser).

**Comment by pvnr0082t** [comment]
We prefer to use npm auto-updater in cdnjs. 

Because the `main` filed in `package.json` will be removed when add this lib into cdnjs, I think it'll not be a problem.

It'll be better if you can publish `dist` to NPM in future versions, or you can keep the structure and we will choose to automatically update this library via GitHub. It's up to you. What do you think?
Thank you.

**Comment by lupomontero** [comment]
Hi @pvnr0082t, I have now released new version (1.1.14) including the `dist` directory:
https://www.npmjs.com/package/psl

Cheers!
